### ISSUE TITLE
Plan release v0.10.0?

### ISSUE THREAD

**Issue by elliot-100** [Olen/Spond#61]
@Olen what do you think about a release to include everything so far, plus maybe PR #56?

I'm happy to do some of the admin but have still never published anthing to PyPI, or looked at how to integrate that into Poetry and or CI.

**Comment by Olen** [comment]
I agree.  I currently have a simple script to create a release, but I'll read up on how to use Poetry.

**Comment by Olen** [comment]
0.10.1 is now released.
(Had to add a .1 to get both github and pypi updated, should work better next time)

**Comment by elliot-100** [comment]
Many thanks @Olen! 🎆
